pkg/types/rekord/v0.0.1/entry.go

```diff
@@ -418,3 +424,45 @@ func (v V001Entry) CreateFromArtifactProperties(ctx context.Context, props types
 
 	return &returnVal, nil
 }
+
+func (v V001Entry) Verifier() (*cx509.Certificate, crypto.PublicKey, openpgp.EntityList, error) {
+	switch f := *v.RekordObj.Signature.Format; f {
```

Comment:
can we grab the `v.keyObj.Verifier` if you implement that common underlying PublicKey interface method?